content/en/tutorials/touch-joypad.md

```diff
@@ -0,0 +1,154 @@
+---
+title: Touchscreen Joypad Controls
+layout: tutorial-page.hbs
+tags: input,ui
+thumb: https://s3-eu-west-1.amazonaws.com/images.playcanvas.com/projects/12/1007506/2F5E56-image-75.jpg
+---
+
+<iframe loading="lazy" src="https://playcanv.as/p/kvE0iJWc/" title="Touchscreen Joypad Controls"></iframe>
+
+[Click here to see the project][project-link].
+
+## Overview
+
+This tutorial shares a Touchscreen Joypad library that can be installed and customized for any project that needs touch screen controls.
+
+We recommend reading about the [PlayCanvas UI system][playcanvas-ui] if you are not familiar with it as it to render and position the controls.
```

Comment:
could be '... as it is used to render ...'?

Reply:
```suggestion
We recommend reading about the [PlayCanvas UI system][playcanvas-ui] if you are not familiar with it as it is used to render and position the controls.
```